William Rudder/week1/MTALAB/js/MTALAB.js

```diff
@@ -0,0 +1,63 @@
+function planTrip(lineA, stopA, lineB, stopB) {
+
+//note: this only stop have intersects all lines is "U.S"
+const lines = {
+
+  "N": ["Times Square", "34th", "28th", "23rd", "Union Square", "8th"],
+
+  "L": ["Times Square", "8th", "6th", "Union Square", "3rd", "1st"],
+
+  "6": ["Grand Central", "33rd", "28th", "23rd", "Union Square", "Astor Place"]
+}
+// create a function that helps us travel form one stop to the other on the same line
+//parameters: line, start stop, end stop
+function stopsOnLine(line, stopA, stopB) {
+  const currentLine = lines[line];
+if (!currentLine) {
+  throw `line ${line} does not exist!`
+}
```

Comment:
your if should be indented here. so: 
```
function stopsOnLine(line, stopA, stopB) {
  const currentLine = lines[line];
  if (!currentLine) {
    throw `line ${line} does not exist!`
  }
```

The indentation makes it so much more readable, so please keep it in mind.